src/ClientHandle.cpp

```diff
@@ -129,7 +129,6 @@ cClientHandle::~cClientHandle()
 			World->RemovePlayer(m_Player, true);  // Must be called before cPlayer::Destroy() as otherwise cChunk tries to delete the player, and then we do it again
 			m_Player->Destroy();
 		}
-		delete m_Player;
```

Comment:
Well, no, I am not sure. `m_Player->Destroy();` will add the player destruction to the queue. The problem I can see is that `m_Player->Destroy();` is inside a conditional "if" so it's not guaranteed to free it. Guess I'll change this and make the player destruct here as usual.